son-gtklic/docs/swagger.yaml

```diff
@@ -0,0 +1,547 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
+swagger: '2.0'
+info:
+  title: Sonata's License Manager
+  description: Manage licenses Sonata services.
+  version: "1"
+# the domain of the service
+host: sonata-vm1.aws.atnog.av.it.pt
+# array of all schemes that your API supports
+schemes:
+  - http
+# will be prefixed to all paths
+basePath: /api/v1
```

Comment:
We have looked for python solutions for using that versioning system, and we found a library that uses the Accept header for versioning. We've never tried it, but we think it might work. We just need to know more or less the content and format of the header, to start implementing.